### DISCUSSION TITLE
Comibine zuztand with reselect library?

### DISCUSSION THREAD

**Discussion by leobar37** [pmndrs/zustand#527] [Q&A]
[Reselect](https://github.com/reduxjs/reselect)

Hi , recommended use this library with zuztand o there a few cosiderations ?

**Comment by dai-shi** [comment]
I haven't seen examples of using zustand and reselect, but let's wait for someone with some experience.

From the library perspective, zustand is unopinionated and using with reselect is totally fine.

(FYI, my personal opinion is using selector only for render optimization is a bad (or not easy for beginners) abstraction, and I developed a reselect alternative: [proxy-memoize](https://github.com/dai-shi/proxy-memoize))
